src/k8s-extension/azext_k8s_extension/partner_extensions/DataProtectionKubernetes.py

```diff
@@ -72,7 +72,9 @@ def Create(
         configuration_settings,
         configuration_protected_settings,
         configuration_settings_file,
-        configuration_protected_settings_file
+        plan_name,
```

Comment:
I think it's accidental - the configuration_protected_settings_file parameter is removed.  Can you add it back?

Reply:
Good catch, thank you.